src/index.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '*.png'
```

Comment:
@spencerjsmall fixed the image error in typescript. Had to define the module inside of `/src` instead of `/server`. Found the solution through this: https://stackoverflow.com/a/57176553/13178719